benchmarks/torchbench_model.py

```diff
@@ -144,6 +144,48 @@
     "hf_T5_generate",
 }
 
+# This list was extracted from PyTorch's repository: benchmarks/dynamo/torchbench.py
+FORCE_AMP_FOR_FP16_BF16_MODELS = {
```

Comment:
These lists feel prone to divergence. Is this how PyTorch does it, too?

Reply:
Yes.

Reply:
Yukio: can you extract these lists from where they are so that we can import them? That will eliminate any maintenance burden from us (i.e. I don't want us to have to manually keep these lists in sync with the ones in Pytorch.)
IIRC you did something similar with the deny list being extracted into a YAML file.

Reply:
Yes. There is a YAML file for skipped models. These models aren't included in that. I guess I could make the change to include these in the PyTorch repo.